docs/index.md

```diff
@@ -1,7 +1,7 @@
 Documentation for older versions:
 
 jib-maven-plugin
-- [v0.1.7](v0.1.7/jib-maven-plugin.md)
+- [v0.1.7](https://github.com/GoogleContainerTools/jib/blob/03d42652e82c543d0a6c6b69bd8827ec8effd82d/docs/index.md)
```

Comment:
Should this link straight to the maven readme instead of the index? (Same with gradle below) https://github.com/GoogleContainerTools/jib/blob/03d42652e82c543d0a6c6b69bd8827ec8effd82d/docs/v0.1.7/jib-maven-plugin.md

Reply:
Sounds good